skillserver/dialog/default.go

```diff
@@ -0,0 +1,16 @@
+package dialog
+
+type Type string
```

Comment:
Based on the wording in the [docs](https://developer.amazon.com/docs/custom-skills/dialog-interface-reference.html#elicitslot) I would call this a `Directive` instead of a `Type`.

Reply:
My thought process was, the Directive is tge entire thing, like with intents and all, its a DialogType or DirectiveType
I created another package so that we could refer to this as dialog.Type
The json also is sending this thing as "type" hence I thought of keeping it as just camelized type